internal/generator/fluentd/conf_test.go

```diff
@@ -143,12 +143,12 @@ var _ = Describe("Testing Complete Config Generation", func() {
   read_from_head "#{if (val = ENV.fetch('JOURNAL_READ_FROM_HEAD','')) && (val.length > 0); val; else 'false'; end}"
 </source>
 
-# Logs from containers (including openshift containers)
+# Logs from containers 
 <source>
   @type tail
   @id container-input
   path "/var/log/pods/*/*/*.log"
-  exclude_path ["/var/log/pods/openshift-logging_collector-*/*/*.log", "/var/log/pods/openshift-logging_elasticsearch-*/*/*.log", "/var/log/pods/openshift-logging_kibana-*/*/*.log", "/var/log/pods/*/*/*.gz", "/var/log/pods/*/*/*.tmp"]
+  exclude_path ["/var/log/pods/openshift-logging_*/*/*.log", "/var/log/pods/*/*/*.gz", "/var/log/pods/*/*/*.tmp"]
```

Comment:
@vparfonov Rethinking the implementation since this might simplify this change .  The segment  before '*.log' is the container name.  Maybe we should just exclude any containers that match what they call the container in the deployment instead of trying to alter how we collect logs for the eventrouter

Reply:
but it will be almost the same that we had before 
```suggestion
  exclude_path ["/var/log/pods/openshift-logging_*/collector/*.log", /var/log/pods/openshift-logging_*/elasticsearch /*.log", /var/log/pods/openshift-logging_*/kibana/*.log", /var/log/pods/openshift-logging_*/loki/*.log", "/var/log/pods/*/*/*.gz", "/var/log/pods/*/*/*.tmp"]
```

Reply:
@jcantrill ˆˆˆ

Reply:
correct it is almost the same.  Per my discussion with @periklis  please make the container segment "loki*"